.gitlab-ci.yml

```diff
@@ -1,7 +1,3 @@
-variables:
-  # Do not modify this - must be the repository name for Kubernetes gitlab runners to run
-  KUBERNETES_SERVICE_ACCOUNT_OVERWRITE: helm-charts
```

Comment:
Env variable is no longer used: https://github.com/DataDog/gitlab-runner-cirel/commit/1efc85c1dbf54c63b8635b5fd60e8d9ba1f19167#diff-d3a55eef9d7c1c66fa9d35e0056f568d3ffa20d77709e1483a4729cab4f8ee9dR171-R174